Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -145,14 +150,14 @@ public void ConfigureAuthServicesForJwtBearer(IServiceCollection services)
                 };
                 o.Events = new JwtBearerEvents()
                 {
-                    OnAuthenticationFailed = c =>
+                    OnAuthenticationFailed = (AuthenticationFailedContext context) =>
```

Comment:
Does this need to be in a lambda? Straight forward code would be more readable.

Reply:
Refactored